manual/moderncv_userguide.tex

```diff
@@ -910,7 +966,21 @@ \section{Implementation details}
 \label{section:implementationDetails}
 
 \subsection{Creating your own styles}
-\todox{Explain how to create styles and and how to recombine headers, footers, bodies etc.}
+Currently {\moderncv} ships with five different styles: \texttt{casual} (default), \texttt{classic}, \texttt{banking}, \texttt{oldstyle} and \texttt{fancy}. Internally, these styles are numbered 1, 2, 3, 4 and 5, respectively. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.
+
+ifferent styles' head, body and foot can be combined to create custom styles. This is done by setting \code{\\moderncvhead}, \code{\\moderncvbody} and \code{\\moderncvfoot} in the preamble. Here is an example of a custom style with a \texttt{banking} head, an \texttt{oldstyle} body and a \texttt{casual} foot:
```

Comment:
```suggestion
Different styles' head, body and foot can be combined to create custom styles. This is done by setting \code{\\moderncvhead}, \code{\\moderncvbody} and \code{\\moderncvfoot} in the preamble. Here is an example of a custom style with a \texttt{banking} head, an \texttt{oldstyle} body and a \texttt{casual} foot:
```

Reply:
Hmm... Somebody stole a `D` here. Thanks! Is now resolved with [e0e9259](https://github.com/moderncv/moderncv/pull/184/commits/e0e9259201620d26f233190f0d366627be653b81).